source/tutorial/rotate-log-files.txt

```diff
@@ -7,111 +7,69 @@ Rotate Log Files
 Overview
 --------
 
-Log rotation using MongoDB's standard approach archives the current
+During normal operation :program:`mongod` and :program:`mongos` report
+a live account of all activity and operations to a log file.
+When the log file reaches its limits, the system must replace or 
+rotate the log file.
+
+MongoDB's standard log rotation approach archives the current
 log file and starts a new one. To do this, the :program:`mongod` or
 :program:`mongos` instance renames the current log file by appending a
-UTC (GMT) timestamp to the filename, in :term:`ISODate` format. It then
+UTC timestamp to the filename, in :term:`ISODate` format. It then
 opens a new log file, closes the old log file, and sends all new log
 entries to the new log file.
 
-MongoDB's standard approach to log rotation only rotates logs
+Alternatively, you can configure MongoDB to use the standard Linux/Unix
+log rotate behavior by setting :setting:`systemLog.logRotate` or
+:option:`--logRotate` to ``reopen``. With ``reopen``, :program:`mongod`
+or :program:`mongos` closes the log file, and
+then reopens a log file with the same name, expecting that another
+process renamed the file prior to rotation.
+
+Finally, you can configure :program:`mongod` to send log data to the ``syslog``. In
+this case, you can take advantage of alternate logrotation tools.
+
+By default, MongoDB only rotates logs
 in response to the :dbcommand:`logRotate` command, or when the
 :program:`mongod` or :program:`mongos` process receives a ``SIGUSR1``
 signal from the operating system.
 
-Alternately, you may configure mongod to send log data to ``syslog``. In
-this case, you can take advantage of alternate logrotation tools.
-
 .. seealso:: For information on logging, see the
    :ref:`monitoring-standard-loggging` section.
 
-Log Rotation With MongoDB
--------------------------
+Log Rotation with ``--logRotate rename``
+----------------------------------------
 
 The following steps create and rotate a log file:
```

Comment:
I know you blurbed above in the overview, but would also be nice to have the descriptions within the sections as well.